apps/web/src/design-system/template-button/ChannelButton.tsx

```diff
@@ -310,7 +310,7 @@ const Dropzone = styled.div<{ dark: boolean }>`
   border: 1px dashed ${({ dark }) => (dark ? colors.B30 : colors.B80)};
 `;
 
-const Button = styled(UnstyledButton)`
+const Button: any = styled(UnstyledButton)`
```

Comment:
Can we remove the `any` part here?

Reply:
This component was leading to errors when I was working on it locally, I took this `any` reference from the subsequent file `TemplateButton.tsx`. After that, the error got resolved.